ext/dapr-ext-workflow/dapr/ext/workflow/workflow_client.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+from datetime import datetime
+from typing import Any, TypeVar, Union
+from dapr.conf import settings
+
+from durabletask import client
+from dapr.ext.workflow.workflow_state import WorkflowState
+from dapr.ext.workflow.workflow_context import Workflow
+
+T = TypeVar('T')
+TInput = TypeVar('TInput')
+TOutput = TypeVar('TOutput')
+
+
+class WorkflowClient:
+
+    def __init__(self, host: Union[str, None] = None, port: Union[str, None] = None):
+        if host is None:
+            host = settings.DAPR_RUNTIME_HOST
+        if port is None:
+            port = settings.DAPR_GRPC_PORT
+        address = f"{host}:{port}"
```

Comment:
Let's raise an error if either `host` or `port` is still `None` even after running through this logic.

Reply:
Updated to set "host" to `localhost`, if DAPR_RUNTIME_HOST is not found.
Updated to raise an error, if DAPR_GRPC_PORT is not found.